docusaurus.config.js

```diff
@@ -113,78 +135,92 @@ module.exports = {
-      async: true
-    }
+      src: "https://semgrep.dev/docs/fs.js",
+      async: true,
+    },
   ],
   presets: [
     [
-      '@docusaurus/preset-classic',
+      "@docusaurus/preset-classic",
       {
         docs: {
-          sidebarPath: require.resolve('./sidebars.js'),
+          sidebarPath: require.resolve("./sidebars.js"),
           // Please change this to your repo.
-          editUrl:
-            'https://github.com/returntocorp/semgrep-docs/edit/main',
-          routeBasePath: '/'
+          editUrl: "https://github.com/returntocorp/semgrep-docs/edit/main",
+          routeBasePath: "/",
         },
         theme: {
-          customCss: require.resolve('./src/css/custom.css'),
+          customCss: require.resolve("./src/css/custom.css"),
         },
         gtag: {
           // You can also use your "G-" Measurement ID here.
-          trackingID: 'G-1851JH9FSR',
+          trackingID: "G-1851JH9FSR",
         },
       },
     ],
     [
-      '@docusaurus/plugin-sitemap',
-        {
-          changefreq: 'weekly',
-          priority: 0.5,
-          trailingSlash: false,
-        },
+      "@docusaurus/plugin-sitemap",
+      {
+        changefreq: "weekly",
+        priority: 0.5,
+        trailingSlash: false,
+      },
     ],
   ],
   plugins: [
     [
-      '@docusaurus/plugin-client-redirects',
+      "@docusaurus/plugin-client-redirects",
       {
         redirects: [
-          { from: "/writing-rules/pattern-logic", to: "/writing-rules/rule-syntax/" },
+          {
+            from: "/writing-rules/pattern-logic",
+            to: "/writing-rules/rule-syntax/",
+          },
           { from: "/writing-rules/index", to: "/writing-rules/overview/" },
-          { from: "/semgrep-ci/managing-policy", to: "/semgrep-app/managing-policy/" },
+          {
+            from: "/semgrep-ci/managing-policy",
+            to: "/semgrep-app/managing-policy/",
+          },
           { from: "/managing-policy", to: "/semgrep-app/managing-policy/" },
           { from: "/integrations", to: "/semgrep-app/notifications/" },
           { from: "/notifications", to: "/semgrep-app/notifications/" },
           { from: "/sso", to: "/semgrep-app/sso/" },
-          { from: "/experiments", to: "/experiments/overview/" },          
+          { from: "/experiments", to: "/experiments/overview/" },
           { from: "/upgrade", to: "/upgrading/" },
           { from: "/semgrep-ci", to: "/semgrep-ci/overview/" },
           { from: "/sample-ci-configs", to: "/semgrep-ci/sample-ci-configs/" },
           { from: "/status/", to: "/language-support/" },
           { from: "/cli-reference/", to: "/cli-usage/" },
           { from: "/ignoring-findings/", to: "/ignoring-files-folders-code/" },
-        ]
-      }
+        ],
+      },
     ],
-  ]
+  ],
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en", "uk"],
+    localeConfigs: {
+      en: {
+        htmlLang: "en-US",
+      },
+    },
+  },
```

Comment:
This is the only part that changed. The rest of the file changed cause my editor auto-formats JS with prettier, the code formatter we use in other repos.